swap_meet/vendor.py

```diff
@@ -1,2 +1,114 @@
+from .item import Item
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory = None):
+        if inventory is None:
+            self.inventory = []
+        else:
+            self.inventory = inventory[:]
+        
+
+    def add(self,item):
+        self.inventory.append(item)
+        return item
+    
+    def remove(self, item):
+        if item not in self.inventory:
+            return False
```

Comment:
👍 Nice approach to look for the case that would prevent us from removing an item and handling that before getting to our main logic. This guards the main logic from a data configuration that would cause a problem.

Note that since this branch `return`s from the function, any following code is effectively the "false" branch. This means the code currently in the false branch could be unindented from beneath the else, and the else removed.

```py
        if item not in self.inventory:
            return False

        # "false" code unindented
        self.inventory.remove(item)
        return item   
```

This approachis very common, so we should get used to expressing our logic this way.